### PR TITLE
CSS Object Model (Houdini), Part IV.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/feature/5682491075592192
https://drafts.css-houdini.org/css-typed-om/

I've also alphabetized the elements in `api/Element.json`. The added property is `computedStyleMap`.

### CODE REVIEW

**Comment by Elchi3** [comment]
The CSS namespace is specified here https://drafts.css-houdini.org/css-typed-om/#numeric-factory.
It lists `ic`, `lh`, `rlh`, `vi`, `vb` which you haven't included here. Is there a reason to omit these?

File: api/CSS.json [hunk 1]
```diff
@@ -0,0 +1,1534 @@
+{
+  "api": {
+    "CSS": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS",
```
Comment (ExE-Boss) [inline api/CSS.json]: This… Doesn’t sound right.

And doesn’t match [the existing table](https://developer.mozilla.org/en-US/docs/Web/API/CSS#Browser_compatibility) or #2167.

Comment (Elchi3) [reply api/CSS.json]: Yeah, the spec mentioned above declares a partial `CSS` namespace with a few properties. The other part of that `CSS` interface is indeed in PR #2167, which was opened earlier, so I'm going to merge that one first and we'll need to rebase here (and take my comment about the missing properties into account).

**Comment by jpmedley** [comment]
"Is there a reason to omit these?" Just an oversight.

**Comment by jpmedley** [comment]
Please take another look.

**Pull request merged**